core/org.eclipse.cdt.core/parser/org/eclipse/cdt/core/dom/parser/cpp/GPPScannerExtensionConfiguration.java

```diff
@@ -246,7 +246,7 @@ public GPPScannerExtensionConfiguration(CompilerType compiler, int version) {
 			addKeyword(GCCKeywords.cp__is_enum, IGCCToken.tTT_is_enum);
 			addKeyword(GCCKeywords.cp__is_final, IGCCToken.tTT_is_final);
 			// __is_floating_point
-			// __is_function
+			addKeyword(GCCKeywords.cp__is_function, IGCCToken.tTT_is_function);
```

Comment:
Note recently GCC added `__is_function` too https://gcc.gnu.org/git/?p=gcc.git;a=commit;h=ffc08431f5c89a20dab21f958bb408d6a604cf54
It is probably safe to add this builtin to GCC above.

Reply:
Thanks. I assume it's going to be in GCC 14.0 based on the roadmap so I added it there.